components/molecules/charts/schedule/helper/columnPagination.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Components - Molecules - Charts - Schedule - Helper - Column Pagination
+ */
+
+// React
+import React from 'react'
+import {
+  addYears,
+  addMonths,
+  addWeeks,
+  addDays,
+  daysToWeeks,
+  format,
+  subWeeks,
+  subDays,
+  subYears,
+  subMonths
+} from 'date-fns'
+
+// Constant
+import THEME_ALIGN from '../../../../constants/align'
+import THEME_SIZE from '../../../../constants/size'
+
+// UI
+import ButtonToolbar from '../../../../atoms/button/toolbar/toolbar'
+import Button from '../../../../atoms/button/button/button'
+import Row from '../../../../atoms/grid/Row'
+
+const args = {
+  align: THEME_ALIGN.Start,
+  size: THEME_SIZE.MD
+}
+
+const calculateCurrent = (currentDate, mode, status = 'current') => {
+  const pCurrentDate = Date.parse(currentDate)
+  switch (mode) {
```

Comment:
@Ojaghi1991 On Switch best practices is to always have a default right. Choose the most common case for the component and make that the default - probably year?

Also the inner conditionals should be switch also - I think switch is always faster than if else